modules/ROOT/pages/shared-resources.adoc

```diff
@@ -4,9 +4,10 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: anypoint studio, shared resources, domains, multiple applications, share ports, domain project
 
-Mule can define selected connectors as common resources and expose them to all apps deployed under a same domain. These resources are known as shared resources. To host them, you must create a Mule Domain Project and then reference it from each of the projects that use the elements in it. Once defined, any Mule app associated with a particular domain can access resources in this file. Note that Mule apps can be associated with only one domain at a time.
+You can define connector configurations as shared resources to expose them to all applications deployed under the same domain. To use shared resources, you create a Mule domain project and then reference it from each of the projects that use the shared resources defined in the domain. After defining the domain, any Mule app associated with the domain can access resources in this file. +
```

Comment:
I think we might be able to get rid of a few words to simplify this, but please do double-check my thinking:
```suggestion
You can define connector configurations to be shared among all applications deployed in one domain. To do so, create a Mule domain project and then reference it from each application. As a result, each app now associated with the Mule domain project can access resources in the domain. +
```
I was a little confused by the original text about apps vs. Mule apps, apps deployed in vs. associated with a domain, domain vs. Mule domain project, apps sharing resources vs. projects using shared resources, and the Mule domain project vs. "this file."
It seemed like the same or similar terms were being used in different ways throughout.
My suggestion might be wrong, but be careful, if you reword this, to be very specific about the differences in these term pairs.